lib/Controller/CallController.php

```diff
@@ -158,7 +159,14 @@ public function downloadParticipantsForCall(string $format = 'csv'): DataDownloa
 
 		fseek($output, 0);
 
-		return new DataDownloadResponse(stream_get_contents($output), 'participants.csv', 'text/csv');
+		// Clean the room name
+		$cleanedRoomName = preg_replace('/-+/', '-', preg_replace('/[\/\\:*?"<>|]/', '-', $this->room->getName()));
+		// Limit to a reasonable length
+		$cleanedRoomName = substr($cleanedRoomName, 0, 100);
+		$date = (new \DateTime())->format('Y-m-d');
+		$fileName = "{$cleanedRoomName} - {$date}.csv";
```

Comment:
Maybe without an additional hyphen?
```suggestion
		$fileName = "{$cleanedRoomName} {$date}.csv";
```